packages/design-system/src/helpers/theme.ts

```diff
@@ -3,11 +3,11 @@ import { DefaultTheme } from 'styled-components';
 import { isKeyOf } from './objects';
 
 export function extractStyleFromTheme<TKey extends keyof DefaultTheme, TSection extends DefaultTheme[TKey]>(
-  themeSection: TSection,
+  themeSection: TSection | null | undefined,
   key: string | number | symbol | undefined,
```

Comment:
Can we do this?

```suggestion
  themeSection?: TSection | null,
  key?: string | number | symbol,
```

Reply:
there is a subtle difference I think, but in this case we can do optional way as well.